SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -144,11 +152,32 @@ void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 
 	TArray<UFunction*> RelevantClassFunctions = SpatialGDK::GetClassRPCFunctions(Class);
 
+	// Save AlwaysWrite RPCs to validate there's at most one per class.
+	TArray<UFunction*> AlwaysWriteRPCs;
```

Comment:
What do you think about moving this check over to when we're generating schema for each class? Currently it'll only fire when you use the class, which for bigger games might not be till well into runtime.

Reply:
That sounds useful! I think doing the check in both places as opposed to moving it would be better, since schema generation isn't required for adding/changing RPCs.